### DISCUSSION TITLE
Pre-populate value for input (not just prompt which then doesn't get submitted)

### DISCUSSION THREAD

**Discussion by clearyphysio** [swiftDialog/swiftDialog#332] [Q&A]
Hi,
I'm presenting a dialog for which I know the user's email address. I want to have an email input field, but also pre-populate it with the value I have. I don't just want it as a prompt, but rather as a pre-filled field (that can be changed).

Is this possible?

Thanks!

**Comment by mm2270** [comment]
Hi. From the Wiki page for text entry:
```
Value

value="<default text>" will pre-populate the textfield with the specified value.

This differs from prompt in that the text can be edited

--textfield "Default Value Test",value="Some default value"
```

**Comment by bartreardon** [comment]
Closing as answered by @mm2270
